corporate/lib/support.py

```diff
@@ -54,6 +57,13 @@ def get_current_plan_data_for_support_view(billing_session: BillingSession) -> P
             if new_plan is not None:
                 plan_data.current_plan = new_plan  # nocoverage
             plan_data.licenses = last_ledger_entry.licenses
-            plan_data.licenses_used = billing_session.current_count_for_billed_licenses()
+            try:
+                plan_data.licenses_used = billing_session.current_count_for_billed_licenses()
+            except MissingDataError:  # nocoverage
+                plan_data.warning = "Recent data missing: No information for used licenses"
```

Comment:
I figure test coverage will be added/done when `current_count_for_billed_licenses` is fully implemented for all billing sessions. Currently, to manually test, I just updated the remote server billing session to raise `MissingDataError` instead of returning 10.